alcs-frontend/src/app/features/application/applicant-info/application-details/parcel/parcel.component.html

```diff
@@ -90,12 +130,6 @@ <h4 [id]="parcel.uuid">Parcel #{{ parcelInd + 1 }}: Parcel and Owner Information
         </div>
         <div>{{ owner.phoneNumber }}</div>
         <div>{{ owner.email }}</div>
-        <div *ngIf="parcel.ownershipTypeCode !== PARCEL_OWNERSHIP_TYPES.CROWN">
-          <a *ngIf="owner.corporateSummary" (click)="openFile(owner.corporateSummary)">{{
-            owner.corporateSummary.fileName
-          }}</a>
-          <app-no-data *ngIf="!owner.corporateSummary"></app-no-data>
```

Comment:
Add this back in. The condition should be removed because that is not relevant, but the anchor link needs to be there for users to open the corporate summary file.